### ISSUE TITLE
Incorrect Weapon Stats

### ISSUE THREAD

**Issue by Xalavar** [Bungie-net/api#743]
I was looking at the different Whisper of the Worm hashes, and I noticed that all of them had the wrong stats. One of them (1891561814, which I assume is supposed to be the correct one) had some correct stats such as Rounds Per Minute, but the Stability in this weapon's hash was 24 while the Reload Speed was 42, despite the [in-game](https://i.gyazo.com/fc0b15db7a576b93980267e234d3d357.png) stats showing that the Stability and Reload Speed stats are the same. 

The other two hashes are 2221042414 and 2221042415.

I might add more weapons if I find any.

**Comment by vthornheart-bng** [comment]
Ah, interesting - thanks for reporting this!

If you look at an actual instance of the item, are the stats correct?

**Comment by LordNazo** [comment]
Sorry for the late reply.

These were the stats of the instanced weapon:

`"stats": {
                    "155624089": {
                        "statHash": 155624089,
                        "value": 42,
                        "maximumValue": 100
                    },
                    "943549884": {
                        "statHash": 943549884,
                        "value": 31,
                        "maximumValue": 100
                    },
                    "1240592695": {
                        "statHash": 1240592695,
                        "value": 86,
                        "maximumValue": 100
                    },
                    "3871231066": {
                        "statHash": 3871231066,
                        "value": 3,
                        "maximumValue": 100
                    },
                    "4043523819": {
                        "statHash": 4043523819,
                        "value": 100,
                        "maximumValue": 100
                    },
                    "4188031367": {
                        "statHash": 4188031367,
                        "value": 42,
                        "maximumValue": 100
                    },
                    "4284893193": {
                        "statHash": 4284893193,
                        "value": 72,
                        "maximumValue": 100
                    }
                }`

**Comment by vthornheart-bng** [comment]
Ah, okay - thanks!  That gives me a good lead to work with.  My suspicion is that the stats are being modified by the Plugs inserted into the item.  In the item definitions, we only return the "base" stats of an item, and don't take into account any plugs that might end up getting inserted once you instantiate it.

However, I'll give it a look when I get the chance - if it's something else going on, we'll be able to see what is up!

**Comment by LordNazo** [comment]
Could it be possible to implement these perk value changes into the "base" stats of the items the perks appear in? It would be much more useful to have minimum and maximum values that reflect the changes these perks make.

**Comment by vthornheart-bng** [comment]
Yes, back in D1 we used to do that and moved away from it in D2: but I think that could be useful!  Let me see about adding that to the backlog on monday.
